tests/test_keras_integration.py

```diff
@@ -274,6 +277,9 @@ def test_push_to_hub(self):
         REPO_NAME = repo_name("PUSH_TO_HUB")
         model = self.model_init()
         model.build((None, 2))
+        x = tf.constant([[0.44, 0.90], [0.65, 0.39]])
+        y = tf.constant([[1, 1], [0, 0]])
+        model.fit(x, y)
```

Comment:
Maybe we should put all of this within a `self.model_init_fitted()` method so you don't need to duplicate code